packages/authorization-platformstate-writer/src/consumerServiceV2.ts

```diff
@@ -360,20 +361,75 @@ export async function handleMessageV2(
         logger
       );
     })
+    .with({ type: "ClientAdminRemoved" }, async (msg) => {
+      const client = parseClient(msg.data.client, msg.type);
+      const pk = makePlatformStatesClientPK(client.id);
+      const clientEntry = await readPlatformClientEntry(pk, dynamoDBClient);
+
+      if (!clientEntry) {
+        logger.info(
+          `Platform-states and token-generation-states. Skipping processing of entry ${pk}. Reason: entry not found in platform-states`
+        );
+        return Promise.resolve();
+      }
+
+      if (clientEntry.version > msg.version) {
```

Comment:
In case of equal version num it's mandatory to update client entry? We can skip also the equal case?